doc/user/flycheck-versus-flymake.rst

```diff
@@ -44,71 +37,68 @@ Please do **not** use this table alone to make your personal judgment.  Read the
++---------------------------+-----------------------+-----------------------+
+|`Multiple syntax checkers  |yes                    |yes                    |
+|per buffer`_               |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Supported languages`_     |>50                    |2 [#]_                 |
+|                           |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|Checking remote files      |said to work, but not  |partly?                |
+|via Tramp                  |officially supported   |                       |
+|                           |[#]_                   |                       |
++---------------------------+-----------------------+-----------------------+
+|`Definition of new         |single declarative     |function definition    |
+|syntax checkers`_          |function/macro         |[#]_                   |
++---------------------------+-----------------------+-----------------------+
+|`Error levels`_            |errors, warnings,      |errors, warnings,      |
+|                           |infos, custom levels   |notes, custom levels   |
+|                           |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Error identifiers`_       |yes                    |no                     |
++---------------------------+-----------------------+-----------------------+
+|`Error parsing`_           |can use arbitrary      |can use arbitrary      |
+|                           |functions; helpers for |parsing functions; no  |
+|                           |regexp, JSON and XML   |built-in helpers       |
++---------------------------+-----------------------+-----------------------+
+|Error highlighting in      |yes                    |yes                    |
+|buffers                    |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|Fringe icons for errors    |yes                    |yes (Emacs 24.1+)      |
++---------------------------+-----------------------+-----------------------+
+|`Error message display`_   |tooltip, echo area,    |tooltip only [#]_      |
+|                           |fully customizable     |                       |
++---------------------------+-----------------------+-----------------------+
+|List of all errors         |yes                    |yes                    |
++---------------------------+-----------------------+-----------------------+
+|`Resource consumption`_    |low                    |high                   |
++---------------------------+-----------------------+-----------------------+
+|`Unit tests`_              |all syntax checkers,   |none?                  |
```

Comment:
Reasonable number of unit tests, both to the core and to the checkers. See under `test/lisp/progmodes/flymake` or something.

Reply:
That's great, I will update this section.